nuxeo-compound-documents/src/test/java/org/nuxeo/compound/documents/CompoundDocumentUtils.java

```diff
@@ -35,50 +35,50 @@
 /** @since 2021.0 */
 public final class CompoundDocumentUtils {
 
+    static final String COMPOUND_DOCTYPE = "CompoundDocument";
+
+    static final String COMPOUND_FOLDER_DOCTYPE = "CompoundDocumentFolder";
+
     private CompoundDocumentUtils() {
         // Utility class
     }
 
-    protected static final String COMPOUND_DOCTYPE = "CompoundDocument";
-
-    protected static final String COMPOUND_FOLDER_DOCTYPE = "CompoundDocumentFolder";
-
-    protected static Blob getTestArchive() throws IOException {
+    static Blob getTestArchive() throws IOException {
```

Comment:
I think we want to keep the protected keyword.

Reply:
Ok I removed them because Intellij warned with :
![image](https://github.com/user-attachments/assets/9e573239-e895-42c8-8772-23669c97efe8)

Reply:
Yes, you can disable this inspection if you don't want to be bothered.

We tend to always have a keyword, to ease reading, but indeed here it is not needed thus Idea warning.